integration-cli/docker_cli_wait_test.go

```diff
@@ -120,3 +121,43 @@ func TestWaitBlockedExitRandom(t *testing.T) {
 
 	logDone("wait - blocking wait with random exit code")
 }
+
+func TestWaitKilledContainerNoWait(t *testing.T) {
+	defer deleteAllContainers()
+
+	runCmd := exec.Command(dockerBinary, "run", "-d", "busybox", "/bin/cat")
+	out, _, err := runCommandWithOutput(runCmd)
+	if err != nil {
+		t.Fatal(out, err)
+	}
+	containerID := strings.TrimSpace(out)
+
+	killCmd := exec.Command(dockerBinary, "kill", containerID)
+	out, _, err = runCommandWithOutput(killCmd)
+	if err != nil {
+		t.Fatal(out, err)
+	}
+
+	done := make(chan struct{})
+	var wg sync.WaitGroup
```

Comment:
Unfortunately I can't see the original context of the outdated diff very easily, so I don't know if it makes sense here. @LK4D4, just for my benefit since I value your opinion, are you saying you consider the "waitgroup to protect goroutine from running out of scope" pattern a bad thing? I'd be interested to know, and be aware of any alternatives :)